shell/platform/darwin/ios/framework/Source/FlutterTextInputPlugin.mm

```diff
@@ -2272,11 +2290,57 @@ - (void)handleMethodCall:(FlutterMethodCall*)call result:(FlutterResult)result {
   } else if ([method isEqualToString:kUpdateConfigMethod]) {
     [self updateConfig:args];
     result(nil);
+  } else if ([method isEqualToString:kOnInteractiveKeyboardPointerMoveMethod]) {
+    double pointerY = [args[@"pointerY"] doubleValue];
+    [self handlePointerMove:pointerY];
+    result(nil);
   } else {
     result(FlutterMethodNotImplemented);
   }
 }
 
+- (void)handlePointerMove:(double)pointerY {
+  double screenHeight = [UIScreen mainScreen].bounds.size.height;
+  double keyboardHeight = _keyboardRect.size.height;
+  // Determines if pointer goes overtop of the keyboard
+  if (screenHeight - keyboardHeight <= pointerY) {
+    // If no screenshot has been taken
+    if (_keyboardViewContainer.subviews.count == 0) {
```

Comment:
Using `_keyboardViewContainer.subviews.count == 0` doesn't seems to be safe and future proof. For example, what if someone added a new subview for `_keyboardViewContainer ` and didn't know this logic exist? Could you use a different indicator?

Reply:
Definitely a concern, the same thing occurs in the tests so there definitely is a need to update this approach. I went through and included a UIView that represents the screenshot. Then we can see if it the superview matches instead.